medium-tests/src/test/java/mediumtest/fixtures/SonarLintBackendFixture.java

```diff
@@ -486,6 +497,11 @@ public SonarLintClientBuilder withSonarProjectBranchMatchingError() {
       this.shouldErrorProjectBranchResolution = true;
       return this;
     }
+
+    public SonarLintClientBuilder withFile(String configScopeId, String fileRelativePath) {
```

Comment:
As mentioned earlier, I am in favor of stopping manually coding this mock, and instead using Mockito.

Reply:
Same answer